.azure-pipelines/templates/configure_azureml_agent.yml

```diff
@@ -16,7 +16,7 @@ steps:
 
       python -m pip install --upgrade pip
       python -m pip install -r .azure-pipelines/requirements/execute_job_requirements.txt
-      python -m pip install promptflow promptflow-tools promptflow-sdk jinja2 promptflow[azure] openai promptflow-sdk[builtins]
+      python -m pip install promptflow>=1.4.0 promptflow-tools promptflow-sdk jinja2 promptflow[azure]>=1.4.0 openai promptflow-sdk[builtins]
```

Comment:
Food for thought, opportunity to generalize and group these 6 dependencies into a separate xxx-requirements.txt, such that it can be referenced by this AzDO YML and by the GHA YML. effectively the 'core' xxx-requirements.txt's that can be shared across CICD platforms.

Reply:
I didn't want to change too much about how they did this in the template but yes I thought it was weird they put these on a separate line and not in a requirements file